npmVowelCounter/index.js

```diff
@@ -0,0 +1,7 @@
+module.exports = (str) => {
+     function getVowels(str) {
```

Comment:
The demo that Monarch did had a refactor of the default export to help future-proof any further development. I _think_ that applies here as well